Solutions/DNS Essentials/NgrokReverseProxyOnNetwork

```diff
@@ -0,0 +1,30 @@
+id: 50b0dfb7-2c94-4eaf-a332-a5936d78c263
+name: Ngrok Reverse Proxy on Network (ASIM DNS Solution)
+description: |
+  'This detection identifies the top 4 Ngrok domains from DNS resolution. Ngrok’s reverse proxy can
+tags:
+  - Schema: ASimDns
+    SchemaVersion: 0.1.6
+requiredDataConnectors: []
+tactics:
+  - Command And Control
```

Comment:
I don't think space is needed here.
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/c8316e08-bad4-45f1-b7df-ae3ea51664ac)